lib/backend/k8s/syncer.go

```diff
@@ -556,16 +571,21 @@ func (syn *kubeSyncer) parseNodeEvent(e watch.Event) *model.KVPair {
 	}
 
 	kvp, err := resources.K8sNodeToCalico(node)
-
 	if err != nil {
 		log.Panicf("%s", err)
 	}
 
+	kvpHostIp := &model.KVPair{
+		Key: model.HostConfigKey{Hostname: node.Name},
```

Comment:
I think so, but the fact that it worked this way worries me a bit, going to rebuild felix/node and fire up another cluster to test